docs/DeveloperGuide.md

```diff
@@ -331,6 +337,54 @@ Extensions:
 - 1a. Tag already exists.
   - 1a1. FoodRem displays error warning to user.
     Use case resumes from step 1.
+  
+#### UC7: Rename a tag
+
+Use Case: UC7 - Rename a tag
+MSS:
+
+1. User chooses to view all tags.
+2. System shows all tags.
+3. User enters the command to rename desired tag.
+4. System informs user that the tag has been updated successfully.
+
+Extensions:
+- 3a. System detects that the new name already exist.
+  - 3a1. System requests for a new tag name that does not exist.
+  - 3a2. User re-enters the command to rename the desired tag.
+  - Steps 3a1-3a2 are repeated until the data entered are correct.
+    Use case resumes from step 4.
+
+
+- 3b. System detects that the name is in an incorrect format.
+  - 3b1. System requests for a new tag name that follows an acceptable format.
+  - 3b2. User re-enters the command to rename the desired tag.
+  - Steps 3b1-3b2 are repeated until the command entered is correct.
+    Use case resumes from step 4.
+
+#### UC8: Removing a tag from an item
+
+Use Case: UC8 - Removing a tag from an item
+Preconditions: User knows the name of the tag they are removing from an item.
+
+MSS:
+1. User enters the command to find the item of interest.
+2. System shows a list containing possible matching items.
```

Comment:
Similarly, possibly edit this in the future to refer to the corresponding use case instead.